tests/common/integration_features/test_suppressions_integration.py

```diff
@@ -628,6 +628,77 @@ def test_supress_by_cve_for_image_scan_with_different_repo_id(self):
         self.assertFalse(suppressions_integration._check_suppression(record3, suppression))
         self.assertFalse(suppressions_integration._check_suppression(record4, suppression))
 
+    def test_suppress_licenses_by_policy(self):
```

Comment:
nice!
I guess suppression by policy is already supported?

Reply:
yes

Reply:
right :)